✨ Add `erp-delivery-note-page-search-extension`-bundle

daniel-rose opened the pull request:
- provide plugin for expanding full text (boosted)
- add main composer changes for new bundle
- add entry to dandelion.json
- ...

julianzimmermann commented on bundles/payone-security-invoice-payment-connector/composer.json:
```diff
@@ -0,0 +1,34 @@
+    "autoload-dev": {
+        "psr-4": {
+            "FondOfOryx\\": "tests/FondOfOryx/",
+            "Generated\\": "src/Generated/"
```
oms is missing

julianzimmermann commented on bundles/payone-security-invoice-payment-connector/codeception.yml:
```diff
@@ -0,0 +1,22 @@
+coverage:
+  enabled: true
+  include:
+    - src/*.php
```
here is missing the standalone and monorepo stuff

julianzimmermann commented on bundles/payone-security-invoice-payment-connector/phpstan.neon:
```diff
@@ -0,0 +1,13 @@
+    scanDirectories:
+        - src/Orm/Zed
+    ignoreErrors:
+        - '#.* on an unknown class Generated\\(Client|Glue|Service|Yves|Zed)\\.*#'
```
needed?

daniel-rose committed:
✨ Add `erp-delivery-note-page-search-extension`-bundle
- provide plugin for expanding full text (boosted)
- add main composer changes for new bundle
- add entry to dandelion.json
- ...

daniel-rose committed:
Merge branch 'main' into feature/erp-delivery-note-page-search-extension

daniel-rose committed:
Merge branch 'main' into feature/erp-delivery-note-page-search-extension
